packages/block-editor/src/components/link-control/search-item.js

```diff
@@ -9,7 +9,44 @@ import classnames from 'classnames';
 import { safeDecodeURI, filterURLForDisplay } from '@wordpress/url';
 import { __ } from '@wordpress/i18n';
 import { Button, TextHighlight } from '@wordpress/components';
-import { Icon, globe } from '@wordpress/icons';
+import {
+	Icon,
+	globe,
+	page,
+	tag,
+	postList,
+	category,
+	file,
+} from '@wordpress/icons';
+
+function SearchItemIcon( { isURL, suggestion } ) {
+	let icon = null;
```

Comment:
Should we have a default fallback icon just in case?

Reply:
I don't think so. The default is just `null` in this case and will just render nothing just like how it used to be.